### PR TITLE
chore: Merge instruction and resource limit pages

### PR DESCRIPTION
**Pull request by dfinity-berestovskyy**

### CODE REVIEW

File: docs/developer-docs/production/resource-limits.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Resource Limits
```
Comment (jessiemongeon1) [inline docs/developer-docs/production/resource-limits.md:1]:
```suggestion
# Resource limits
```

File: docs/developer-docs/production/resource-limits.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| Wasm heap size                                            | 4GB                       |
+| Wasm stable memory                                        | 96GB                      |
+| Wasm custom sections per subnet                           | 2GB                       |
+| Wasm custom sections per canister                         | 1MB                       |
+| Wasm custom sections per canister                         | 16 sections at most       |
+| Wasm code section                                         | 10MB                      |
+| Query calls execution threads                             | 2 per replica node        |
+| Update calls execution threads                            | 4 per subnet              |
+
+Notes:
```
Comment (jessiemongeon1) [inline docs/developer-docs/production/resource-limits.md:31]:
```suggestion

```

File: docs/developer-docs/production/resource-limits.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+1. The limit may be deprecated at some point.
```
Comment (jessiemongeon1) [inline docs/developer-docs/production/resource-limits.md:33]: What does this refer to?

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/developer-docs/production/resource-limits.md (hunk 3 above)
Comment (dfinity-berestovskyy) [reply docs/developer-docs/production/resource-limits.md]: I moved the note back to the table, PTAL.

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/developer-docs/production/resource-limits.md (hunk 3 above)
Comment (jessiemongeon1) [reply docs/developer-docs/production/resource-limits.md]: LGTM! Thanks!

**Pull request merged**